packages/parser/src/constants/bhaiLangSpec.ts

```diff
@@ -89,7 +89,7 @@ export const SPEC = [
   { regex: /^\bagla dekh bhai\b/, tokenType: TokenTypes.AGLA_DEKH_BHAI },
 
   // Number
-  { regex: /^-?\d+/, tokenType: TokenTypes.NUMBER_TYPE },
+  { regex: /^-?[\d.]+/, tokenType: TokenTypes.NUMBER_TYPE },
```

Comment:
This REGEX is wrong. It also detects numbers with multiple dots as float.

Use this REGEX instead : /[+-]?([\d]*[.])?[\d]+/

 
<img width="668" alt="Screenshot 2022-04-05 at 7 58 24 AM" src="https://user-images.githubusercontent.com/24372410/161667254-d283619f-ecca-43dd-8b6b-9020738cb867.png">